pypit/armsgs.py

```diff
@@ -153,17 +154,26 @@ def close(self):
         """
         Close the log file and QA PDFs before the code exits
         """
+        from pypit import arqa
         # Close PDFs
```

Comment:
I removed the closing of the PDF
But the import is needed to auto-make the HTML files.